<?xml version="1.0"?>
<launch>
	<node pkg="gpar_nanook" type="gpar_nanook_node" name="gpar_nanook_node" output="screen" >
	<!-- node parameters -->
	<param name="serial_port" type="string" value="/dev/ttyACM0"/>
	<param name="publish_odom" type="bool" value="true" />
	</node>


	<node pkg="gpar_nanook" type="gpar_nanook_gps" name="gpar_nanook_gps" >
			<param name="serial_port" type="string" value="/dev/ttyUSB0" />
	</node>

</launch>
